app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
```

Comment:
Do we need to add a Welcome controller for creating the navbar?

Reply:
Changed to Listing controller

Reply:
Do we need a controller at all to display a navbar? I don't understand... 😳

Reply:
we deleted that further down